packages.html

```diff
@@ -0,0 +1,334 @@
+  <li>The development of the package is being driven by a research topic, and includes input from both researchers and software developers.</li>
+  <li>The end goal should be one or more peer-reviewed journal publications.</li>
+  <li>A "sunset" date should be communicated on the README and package startup message. This is an indication of how far in the future the work for this package is currently supported.</li>
+  <li>Emphasis should still be placed on testing key features and good documentation during development.</li>
+  <li>Since the goal is a published article, the expectation should be that outside users will use this package to test the reproducibility of the primary author’s data analysis, as well as reproducing the analysis on new data.</li>
+  <li>During development, the package version should remain below v1.0.0.</li>
+</ul>
+</div>
+
+<p>It is important to clearly communicate to any known or unknown users the risk they take with using a package that is in the active research development phase. See <code>?packageStartupMessage</code> to learn how to add a message to that gets displayed whenever a user loads the package.</p>
+      
+      <h3>Example Active Research Packages</h3>
+  <div id="wrapper" class="grid clearfix">
+      <div id="repoContainer">
+          <a href="https://github.com/USGS-R/streamMetabolizer" target="_blank">
+            <div class="repos researchColor">
+            	<h2>streamMetabolizer</h2>
+                <p class="repoDescription">Uses inverse modeling to estimate aquatic metabolism from time series data on dissolved oxygen, water temperature, depth, and light.</p>
+                <p class="underDevelopment">Research</p>
+            </div>
+          </a> 
+          
+          <a href="https://github.com/USGS-R/toxEval" target="_blank">
+            <div class="repos researchColor">
+            	<h2>toxEval</h2>
+                <p class="repoDescription">Initial code for studying ToxCast data in relation to measured.</p>
+                <p class="underDevelopment">Research</p>
+            </div>
+          </a> 
+          
+          <a href="https://github.com/USGS-R/loadflex" target="_blank">
+            <div class="repos researchColor">
+            	<h2>loadflex</h2>
+                <p class="repoDescription">Models and Tools for Watershed Flux Estimates.</p>
+                <p class="underDevelopment">Research</p>
+            </div>
+          </a>
+        
+      </div>
+  </div>
+      
+              
+              <h3>Archive</h3>
+<p>A package with the designation “Archive Research” has several key features:</p>
+
+<div id="plain">
+<ul>
+  <li>Ideally, the development of the active research package was completed with the publishing of a scientific, peer-reviewed journal or report</li>
+  <li>A reproducible work-flow is included either within a vignette or via the final product (such as the journal article)</li>
```

Comment:
should there also be a bullet suggesting that the maintainer provide a `devtools::session_info()` output for a session environment in which the package was successfully used? i don't want to step too much into the realm of model archiving, but we know that dependencies can be a big deal...